src/2025h1/all-hands.md

```diff
@@ -0,0 +1,92 @@
+- Have a social and informal "pre-all hands day" on Rust's 10th birtday: May 15, 2025.
+- Have a two-day all-hands on May 16 and May 17, 2025.
+
+### The "shiny future" we are working towards
+
+The immediate goal is a very succesful and productive Rust All-Hands 2025.
+Hopefully, this will be the first step twoards the larger goal of having regular Rust All-Hands again.
+
+We should be able to use the feedback and lessons learned from this event for the next ones.
+Repeating an event tends to be much easier than organising one from scratch.
+
+## Design axioms
+
+- Accessibility. Ideally, everyone in the project should be able to attend the Rust All-Hands
+- Productivity and effectivity. We should optimally make use of the event, to make it worth everyone's time.
+- Low stress. The event or the planning of it should not burn anyone out. It is a tool to help the project and its members, after all!
+- Space for social events. The goal is not just to work on technical things together, but also to get to know each other and become a closer team.
+
+## Ownership and team asks
+
+**Owner:** Mara
+
+| Subgoal                                        | Owner(s) or team(s)            | Notes |
+| ---------------------------------------------- | ------------------------------ | ----- |
+| Decide to have another Rust All-Hands!         | ![Team][] [leadership-council] | Done! |
+| Pick the dates                                 | RustNL, leadership-council     | Done! |
+| Allocate funds for event                       | ![Team][] [leadership-council] | Done! |
+| Allocate funds for travel                      | Rust Foundation                | Done! |
+| Book the venue                                 | RustNL                         | Done! |
+| Catering / snacks / food / drinks              | RustNL                         | Done! |
+| [Register] for the Rust All-Hands 2025         | *every project member*         | Majority already signed up! |
+| Send out conformations/tickets                 | Mara                           |       |
+| Send out detailed information                  | Mara                           |       |
+| Answer logistical questions                    | Mara                           |       |
+| Interface between project and RustNL           | Mara                           | (Mara is part of both Rust and RustNL.) |
+| Make hotel reservations                        | RustNL                         | In progress |
+| Book hotel                                     | *all participants*             | RustNL will provide suggestions |
+| Book travel                                    | *all participants*             |       |
+| Team swag                                      |                                |       |
+| ↳ Decide on what to get (stickers? shirts?)    | ![Team][] [leadership-council] | Suggestions very welcome! |
+| ↳ Acquire it                                   | RustNL                         |       |
```

Comment:
I'm glad this made the list! Given that there will be a time constraint to make sure things are ordered and arrive in time, someone should start figuring this out sooner rather than later to be sure we don't miss the deadline.